portfolio/src/main/webapp/blog-script.js

```diff
@@ -9,4 +9,66 @@ function openNav() {
 function closeNav() {
   document.getElementById("mySidebar").style.width = "0";
   document.getElementById("main").style.marginLeft= "0";
+}
+
+google.charts.load('current', {
+    'packages':['geochart'],
+    // Note: you will need to get a mapsApiKey for your project.
+    // See: https://developers.google.com/chart/interactive/docs/basic_load_libs#load-settings
+    'mapsApiKey': 'AIzaSyD6AxuefQv6sbT-WwW9bGDnoEacGtLDxvk'
+});
+
+google.charts.setOnLoadCallback(drawRegionsMap);
+
+function drawRegionsMap() {
+var data = google.visualization.arrayToDataTable([
+    ['Country', 'Popularity'],
+    ['Germany', 200],
+    ['United States', 300],
+    ['Brazil', 400],
+    ['Canada', 500],
+    ['France', 600],
+    ['RU', 700]
+]);
+
+var options = {};
+
+var chart = new google.visualization.GeoChart(document.getElementById('regions_div'));
+
+chart.draw(data, options);
+}
+
+google.charts.load('current', {packages:["orgchart"]});
+    google.charts.setOnLoadCallback(drawChart);
+
+    function drawChart() {
+    var data = new google.visualization.DataTable();
+    data.addColumn('string', 'Name');
+    data.addColumn('string', 'Manager');
+    data.addColumn('string', 'ToolTip');
+
+    // For each orgchart box, provide the name, manager, and tooltip to show.
+    data.addRows([
+        [{'v':''},
+           'Grandma Patterson', 'The President'],
+           
+           [{'v':'Grandma Patterson'},
+           '', 'The President'],
+           
+          [{'v':'Jim'},
+           'Mike', 'VP'],
+          ['Alice', 'Mike', ''],
+          
+               [{'v':'Joe'},
+           'Grandma Patterson', 'VP'],
+          ['Ally', 'Grandma Patterson', ''],
+          
+          ['Bob', 'Jim', 'Bob Sponge'],
+          ['Carol', 'Bob', '']
+        ]);
```

Comment:
maybe you could add comments here or something explaining what is going on because it isn't super clear. It seems to me like you are using an org chart to simulate a family tree but I don't think i'd be able to discern that if I hadn't seen the end result.

Reply:
Will do, I will make sure to add comments to my next PR.